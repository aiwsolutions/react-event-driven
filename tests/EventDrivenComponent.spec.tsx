import * as React from 'react';
import * as renderer from 'react-test-renderer';
import EventDrivenComponent from '../src/EventDrivenComponent';

test('<EventDrivenComponent> - should render ok', () => {
    const comp = renderer.create(
        <EventDrivenComponent />,
    );
    expect(comp).toMatchSnapshot();
});
