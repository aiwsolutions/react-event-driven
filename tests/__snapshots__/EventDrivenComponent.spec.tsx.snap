// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EventDrivenComponent> - should render ok 1`] = `null`;
